html/browsers/windows/auxiliary-browsing-contexts/opener-setter.window.js

```diff
@@ -0,0 +1,39 @@
+[
+  undefined,
+  42,
+  function() { return "hi" },
+  "hi",
+  {},
+  [],
+  Symbol()
+].forEach(val => {
+  const title = typeof val === "symbol" ? "Symbol" : String(val);
```

Comment:
`String(Symbol())` works too, so you could just set title to `String(val)` unconditionally.